docs/src/configuration/routes/https.md

```diff
@@ -34,6 +37,15 @@ TLS is a newer encryption system that has replaced SSL, but the name SSL is stil
 In practice, they mean the same thing today, but TLS is the more correct term.
 {{</ note >}}
 
+### Limitations
+
+{{% lets_encrypt_limitations %}}
+
+If you need more hostnames than that, obtain additional certificates or a wildcard certificate from another TLS provider. 
+Alternatively, consider splitting your project up into multiple Platform.sh projects.
+
+Your hostnames, for which the certificate is being generated, need to be available for the [DNS Challenge](#dns-challenge).
```

Comment:
It's not really a limit, but I thought it as a reminder about the part below:
> By default, Platform.sh checks that both the `some-example.platform.sh` and `www.some-example.platform.sh` domains are pointing to your project.

Since some customers do not add the `www` subdomain to point to the server, which causes the auto-update of the certificate to fail and then also makes the live site unavailable.

Reply:
That might be why the docs certificate generation was failing a while back :grinning: 

I wouldn't say it's a limit, though, so it's slightly out of place here. Since this PR isn't about restructuring the article, I'd recommend taking it out for now and revisiting it later. :shrug:

Reply:
That's true, I removed it for now. For the next rework I think we might want to point it out since it can be a bad surprise for the customer than may have an impact on production.